L2-Zad.2-Projekt_Maven/pom.xml

```diff
@@ -0,0 +1,16 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
To chyba ne idotyczy tego zadania?

Reply:
No ok, ale ja nie mam pojęcia jak to zrobić żeby się nie dodawały zmiany z poprzedniego zadania, które zostały już wprowadzone, ale nie zostały zaliczone i zmergowane.

Reply:
kazde nowe zadanie zaczynasz od checkoutu na branch master. Na branchu master robisz update project (w intelliju) lub poprzez console git pull. Nastepnie tworzysz nowy branch i sie na niego checoutujesz, robisz zadanko, commit, push i wystawiasz Pull Requesta. Dopiero po otrzymaniu approva mergujesz branch z zadaniem do mastera. Jesli oczekujesz na akceptacje w mozesz robic kolejne zadania ale zaczynasz zawsze od checoutu do mastera

Reply:
aa ok, ja nie zcheckoutowałem do mastera. W ogóle jak przeszedłem z konsoli na robienie tego w Intellij to się trochę pogubiłem, zobaczymy co mi się uda wysłać za następnym razem :)